hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/operations/MemberAttributeChangedOp.java

```diff
@@ -30,12 +30,12 @@
 
     private MemberAttributeOperationType operationType;
     private String key;
-    private Object value;
+    private String value;
```

Comment:
You can use `writeUTF/readUTF` when serializing.

Reply:
good catch! thanks